src/notification.c

```diff
@@ -282,6 +282,29 @@ void notification_init(notification *n)
         n->body     = n->body     ? n->body     : g_strdup("");
         n->category = n->category ? n->category : g_strdup("");
 
+        /* log */
+        if (settings.log_notifications) {
```

Comment:
Sorry, but indentation is 8 spaces.

Reply:
Oh, and of course: Why do you do this stuff **before** sanitation!?

Reply:
And thirdly, there's `notification_format_message`, which does the job with a few modifications much better IMHO.

Reply:
Oo nice.